src/lib/modules/response/resp_delay.go

```diff
@@ -57,8 +58,13 @@ func HandleGetRequest(url string) utils.Response {
 	// Time init
 	start := time.Now().UnixNano()
 	resp := *utils.SendGETRequest(url)
+	defer resp.Body.Close()
+	content, err := ioutil.ReadAll(resp.Body)
 
-	resLength := resp.ContentLength
+	if err != nil {
+		panic(err)
+	}
+	resLength := int64(len(content))
```

Comment:
The default output of `len()` is `int`. So, even if we increase the output range to a 64-bit like here, the output will be the range of 32-bit only, but wasting the extra 32-bit since the extra range is of no use. So, we should rather keep this to `int` type only.

Reply:
The response length stored in the structure type `response`, has a key with `ResLength` of type int64, so I converted the value to type int64. If changing the type of the element will be fine, I will update that.

Reply:
Yeah sure. We can change the type of the struct.